letsencrypt/client/tests/acme_util.py

```diff
@@ -66,3 +69,76 @@ def gen_combos(challs):
     # Gen combos for 1 of each type, lowest index first (makes testing easier)
     return tuple((i, j) if i < j else (j, i)
                  for i in dv_chall for j in cont_chall)
+
+
+def chall_to_challb(chall, status):  # pylint: disable=redefined-outer-name
+    """Return ChallengeBody from Challenge."""
+    kwargs = {
+        "chall": chall,
+        "uri": chall.typ+"_uri",
+        "status": status,
+    }
+
+    if status == "valid":
```

Comment:
it would be saner to use constants, i.e. `messages2.STATUS_VALID` here, and in any `chall_to_challb` call